docs/source/api/aihwkit.nn.modules.conv.rst

```diff
@@ -5,4 +5,3 @@ aihwkit.nn.modules.conv module
    :members:
    :undoc-members:
    :show-inheritance:
-   :exclude-members: randn, zeros, arange, cat, ones
```

Comment:
Why is this line always deleted? Is this not needed anymore?

Reply:
it is not needed anymore.